### PR TITLE
ICMP Check #15

### PR DESCRIPTION
**Pull request by sainivikas**
Regarding https://github.com/flanksource/canary-checker/issues/15

### CODE REVIEW

File: fixtures/http_pass.yaml [hunk 1]
```diff
@@ -4,4 +4,10 @@ http:
     thresholdMillis: 3000
     responseCodes: [201,200,301]
     responseContent: ""
-    maxSSLExpiry: 60
\ No newline at end of file
+    maxSSLExpiry: 60
+icmp:
```
Comment (moshloop) [inline fixtures/http_pass.yaml]: I think remove this from the http_pass file 
+ add a failing icmp check

Comment (sainivikas) [reply fixtures/http_pass.yaml]: @moshloop removed. icmp_fail.yaml added

File: checks/icmp.go [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+        }
+    }
+    return result
+}
+
+func (c *IcmpChecker) checkICMP(urlObj pkg.URL, packetCount int) (*pkg.ICMPCheckResult, error) {
+    ip := urlObj.IP
+    pinger, err := ping.NewPinger(ip)
+    if err != nil {
+        //fmt.Println("ERROR: %s\n", err.Error())
```
Comment (moshloop) [inline checks/icmp.go]: remove commented out code

File: checks/icmp.go [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+        return nil, err
+    }
+    pinger.Count = packetCount
+    pinger.Timeout = time.Second * 10       // 10 seconds
```
Comment (moshloop) [inline checks/icmp.go]: remove unnecessary comment

File: checks/icmp.go (hunk 2 above)
Comment (sainivikas) [reply checks/icmp.go]: @moshloop Done

File: checks/icmp.go (hunk 3 above)
Comment (sainivikas) [reply checks/icmp.go]: @moshloop Done

**Pull request merged**